### DISCUSSION TITLE
thinking about some new fatures

### DISCUSSION THREAD

**Discussion by davidputh** [anirudhmalik/xhunter#107] [Ideas]
Dear Developers,
I think it will be better if the app icon would hide.
Then keylogger , install and uninstalling and some other features from msfvenom android/reversed_tcp would be very useful .

**Comment by anirudhmalik** [comment]
Closing in favor of v2. See [v2.0-demo](https://github.com/anirudhmalik/xhunter/releases/tag/v2.0-demo) and the repo README. New bugs: open a fresh issue with repro steps. Thanks.
